code/modules/codex/entries/guns_codex.dm

```diff
@@ -146,6 +146,16 @@
 		traditional chargers and their capability to switch their lens, allowing more flexibility, something that a ballistic weapon \
 		aren't capable of."
 
+/datum/codex_entry/plasma_weapons
```

Comment:
what is this point of adding a codex entry if you haven't listed them yet in the generic plasma weapon code?